src/Parsers/ExpressionListParsers.cpp

```diff
@@ -1090,6 +1096,8 @@ class ArrayElementLayer : public BaseLayer<TokenType::Comma, TokenType::ClosingS
 class CastLayer : public Layer
 {
 public:
+    CastLayer() : Layer(true, true) {}
```

Comment:
```suggestion
    CastLayer() : Layer(/*allow_alias*/ true, /*allow_alias_without_as_keyword*/ true) {}
```
also in other constructors